Data/member.json

```diff
@@ -500,15 +500,15 @@
             "githuburl": "https://github.com/iamzaidsoomro",
             "linkedinurl": "https://www.linkedin.com/in/zaid-ahmed-742b77200/"
         },
-      {
+        {
             "name": "Aleksey-Voko",
             "avatar": "https://avatars.githubusercontent.com/u/61884038?v=4",
             "description": "Python, Java, Kotlin, Android, Go",
             "githuburl": "https://github.com/Aleksey-Voko",
             "linkedinurl": ""
         }
         ,
```

Comment:
Please can you move the comma to be on line 509, and after that it should be fine @Abhi6722

Reply:
Done @CodingSpecies